astroquery/gaia/tests/test_gaiatap.py

```diff
@@ -370,31 +370,32 @@ def test_load_data(self):
         band = None
         format = "votable"
         verbose = True
+        data_structure = "INDIVIDUAL"
+        output_file = os.path.abspath("output_file")
+        if not output_file.endswith("gaia/test/output_file"):
+            output_file = os.path.abspath("gaia/test/output_file")
```

Comment:
we need to be more os agnostic, so please use os.path.join instead hard coded separators